source/nodejs/adaptivecards-designer/src/card-designer-surface.ts

```diff
@@ -839,6 +843,32 @@ export class CardDesignerSurface {
         }
     }
 
+    // Outstanding issue: cannot drop an element into a ColumnSet
+    // Find the index of the designer surface where the dragged peer should be dropped
+    calculateDraggedPeerIndex(currentRoot: DesignerPeers.DesignerPeer): boolean {
```

Comment:
nit: the name doesn't seem to match what it does.

it returns a boolean value, but the method name indicates it returns an integral value.